docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -60,14 +72,16 @@ following limitations currently exist for SOA Suite domains:
   currently supports the WebLogic MBean trees only.  Support for JRF MBeans has not
   been added yet.
 
+{{% notice note %}}
+For early access customers, with bundle patch access, it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patches installed. However the customers who do not have access to the bundle patches can obtain the Oracle SOA Suite Docker image from `container-registry.oracle.com` as described in below section.
```

Comment:
it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). -> we recommend that you build and use the Oracle SOA Suite Docker image with the latest bundled patches for Oracle SOA and Oracle Service Bus (OSB).

Reply:
Except bundle vs bundled okay to change.

Reply:
Agreed.